packages/markdown-docx/src/ToCiceroMarkVisitor.js

```diff
@@ -305,17 +332,20 @@ class ToCiceroMarkVisitor {
      * @param {Array}   node              Node to be traversed
      * @param {string}  calledBy          Parent node class that called the function
      * @param {object}  nodeInformation   Information for the current node
-     * @returns {string} Value in <w:t> tags
+     * @returns {string} NodeInformation array if optional else Value in <w:t> tags
```

Comment:
What exactly are the contents of the array if node is optional? You can encode that with `{string}` by using a union operator.